src/injected/visualization/accessible-names-formatter.ts

```diff
@@ -40,11 +40,9 @@ export class AccessibleNamesFormatter implements Formatter {
                 text: formattedAccessibleName,
                 fontWeight: '400',
                 fontSize: '10px',
-                outline: '3px dashed',
             },
             outlineColor: AccessibleNamesFormatter.style.outlineColor,
             outlineStyle: 'dashed',
-            outlineWidth: '3px',
```

Comment:
same as above

Reply:
outlineWidth property was removed from its interface DrawerConfiguration in its previous commits. Tried checking in UI but could not find the relevant page for the property.